microproxy/test/layer/test_socks.py

```diff
@@ -194,7 +180,7 @@ def test_handle_connection_timeout(self):
         self.assertIsInstance(self.event, Response)
         self.assertEqual(self.event.status, RESP_STATUS["NETWORK_UNREACHABLE"])
         self.assertEqual(self.event.atyp, ADDR_TYPE["IPV4"])
-        self.assertEqual(self.event.addr, "1.2.3.4")
+        self.assertEqual(str(self.event.addr), "1.2.3.4")
```

Comment:
Same problem, cast the variable that we want to measure make it's unknown that what's the actual type of the variable.